pkgs/games/hyperrogue/default.nix

```diff
@@ -1,20 +1,27 @@
-{ lib, stdenv, fetchFromGitHub, SDL, SDL_ttf, SDL_gfx, SDL_mixer, autoreconfHook,
-  libpng, glew, makeDesktopItem }:
+{ lib, stdenv, fetchFromGitHub, SDL, SDL_ttf, SDL_gfx, SDL_mixer, libpng,
+  glew, dejavu_fonts, makeDesktopItem }:
 
 stdenv.mkDerivation rec {
   pname = "hyperrogue";
-  version = "11.3o";
+  version = "12.0u";
 
   src = fetchFromGitHub {
     owner = "zenorogue";
     repo = "hyperrogue";
     rev = "v${version}";
-    sha256 = "0bijgbqpc867pq8lbwwvcnc713gm51mmz625xb5br0q2qw09nkyh";
+    sha256 = "sha256-zG8Z+wpwr5z2r2CcjNxKOdiqXUN0AFChZcihUWel68A=";
   };
 
-  CPPFLAGS = "-I${lib.getDev SDL}/include/SDL";
+  CXXFLAGS = [
+    "-I${lib.getDev SDL}/include/SDL"
+    "-DHYPERPATH='\"${placeholder "out"}/share/hyperrogue/\"'"
+    "-DRESOURCEDESTDIR=HYPERPATH"
+    "-DHYPERFONTPATH='\"${dejavu_fonts}/share/fonts/truetype/\"'"
```

Comment:
```suggestion
    "-DHYPERPATH=${placeholder "out"}/share/hyperrogue/"
    "-DRESOURCEDESTDIR=HYPERPATH"
    "-DHYPERFONTPATH=${dejavu_fonts}/share/fonts/truetype/"
```

Reply:
These quotes are actually necessary; the macros need to expand to string literals or something else that would be valid C code.